back-end/app/Http/Controllers/User/UsersController.php

```diff
@@ -170,4 +170,25 @@ public function logout(Request $user_id)
             'message' => 'Logged out',
         ], 201);
     }
+
+    // Search users by name
+    public function search($name)
+    {
+        $fullName = explode(" ", $name);
+
+        $firstName = $fullName[0];
+        $lastName = $fullName[count($fullName) - 1];
+
```

Comment:
What if I want to look for the lastname and just inputted 1 word?
or my input is the lastname first then firstname. The user don't know the algorithm so better use the $name query for both firstname and lastname.

Reply:
Ay sir I have the changes of this section in other PR.

Reply:
Ay sir I have the changes of this section in other PR.

Reply:
I added a logic at the bottom with that already I think it's in the activity branch.
![image](https://user-images.githubusercontent.com/104751512/174544850-e9d9e903-fc17-4d72-88a5-edaac1fd256e.png)